open_schools_platform/history_management/paginators.py

```diff
@@ -0,0 +1,7 @@
+from rest_framework.pagination import PageNumberPagination
+
+
+class ApiHistoryListPagination(PageNumberPagination):
+    page_size = 200
+    page_size_query_param = 'page_size'
+    max_page_size = 500
```

Comment:
Is 500 a good value?
What if several hosts would request 500 histories in one moment?
Do we handle it?

Reply:
I don't know